packages/app/src/specs/SpecsList.vue

```diff
@@ -11,6 +11,13 @@
       class="pb-32px"
       :result-count="specs.length"
       @newSpec="showModal = true"
+      @spec-pattern="bindingsOpen = true"
+    />
+    <SpecPatternModal
+      v-if="props.gql.currentProject && bindingsOpen"
```

Comment:
Should we combine the `v-if` and `:show`? It looks like they do same thing (conditionally hide/show?)

Reply:
I think they have different purposes here. It might be better to make the v-if depend just on `props.gql.currentProject` so that the SpecPatternModal component renders only once, when there is a `currentProject`, and keep `bindingsOpen` to only manage the open/closed state. With the current v-if we add/remove the component from the DOM, at the same time as we are opening and closing the modal.

Reply:
@marktnoonan I'll implement what you've stated above, and also rename some variables as `bindingsOpen` was a copy and paste and it should be named more contextually.